### PR TITLE
change color of select arrow to black

### PR DESCRIPTION
**Pull request by jmophoto**

### CODE REVIEW

File: src/components/Select.js [hunk 1]
```diff
@@ -212,7 +212,7 @@ const styles = {
     position: 'relative'
   },
   downArrow: {
-    color: '#999999',
+    color: '#000000',
```
Comment (bsbeeks) [inline src/components/Select.js]: This should probably inherit the same color as the `selectedStyle`:
```
color: selectedStyle.color || '#000000'
```

Comment (jmophoto) [inline src/components/Select.js]: Since selectedStyle is a prop, can I access it outside the Select object, though?

Comment (bsbeeks) [inline src/components/Select.js]: No, so we'll need to merge the prop color onto the down arrow element here: https://github.com/jmophoto/mx-react-components/blob/jmo/select-icon-color/src/components/Select.js#L175. Something like this:
```
style={objectAssign(styles.downArrow, { color: this.props.selectedStyle.color || '#000000' })}
```

**Comment by bsbeeks** [comment]
:shipit: Will merge with next release.

**Pull request merged**